### DISCUSSION TITLE
Route [filament.pages.dashboard] not defined.

### DISCUSSION THREAD

**Discussion by Temian1** [filamentphp/filament#2505] [Help]
### Package

filament/filament

### Package Version

2

### Laravel Version

v8

### Livewire Version

2

### PHP Version

8

### Bug description

i installed a new laravel breeze project and i tried login in to the admin dashboard not define

**Comment by danharrin** [comment]
Cannot reproduce.

**Reply by saineshmamgain** [reply]
@danharrin I am also getting this issue. What I did was changed `path` to `/` in filament config file.

**Reply by danharrin** [reply]
Please open an issue with a reproduction repo
